src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2423,12 +2436,48 @@ def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certifi
         logger.warning(lnk_msg)
 
     kv_cert_def = Certificate(location=location, key_vault_id=kv_id, password='',
-                              key_vault_secret_name=key_vault_certificate_name, server_farm_id=server_farm_id)
+                              key_vault_secret_name=kv_secret_name, server_farm_id=server_farm_id)
 
     return client.certificates.create_or_update(name=cert_name, resource_group_name=resource_group_name,
                                                 certificate_envelope=kv_cert_def)
 
 
+def get_import_kv_details(cmd, client, certificate_name, key_vault):
+    kv_id = None
+    kv_secret_name = certificate_name
+    # if key_vault is not populated, we are attempting to import an App Service Certificate
+    if not key_vault:
+        certs = client.app_service_certificate_orders.list()
```

Comment:
we are not passing any subscription or other parameters here - so what does this list do?

Reply:
Based on your comment below, this is going to cause a few issues, we might have to make a new client if the ASC is in a different susbcription, will take a look